tests/pact/keysets.repl

```diff
@@ -125,3 +125,50 @@
 
 (env-keys ["c"])
 (enforce-keyset 'rotate-me)
+
+(begin-tx)
+
+
+;;; test caps
+
+(env-data { "a": ["a"], "ab": { "keys": ["a","b"], "pred": "keys-any" } })
+
+(module sig-caps G
+  (defcap G () true)
+
+  (defconst KEYSET_A (read-keyset "a"))
+  (defconst KEYSET_AB (read-keyset "ab"))
+
+  (defcap A (x:string y:bool)
+    (enforce-keyset KEYSET_A))
+
+  (defcap AB ()
+    (enforce-keyset KEYSET_AB))
+
+  (defun run-A (x:string y:bool)
+    (with-capability (A x y) true))
+
+  (defun run-AB ()
+    (with-capability (AB) true))
+  )
+
+(env-keys ["a"])
+(expect "A succeeds with unscoped sig" true
+        (run-A "foo" false))
+(expect "AB succeeds with unscoped A sig" true
+        (run-AB))
+
+(env-sigs [{ 'key: "a", 'clist: [(A "bar" true)] },
```

Comment:
It would be useful to update the keyset expression documentation (https://pact-language.readthedocs.io/en/latest/pact-reference.html#keyset) to show the addition of `clist`. And provide an examples of it and what they mean.
Also, should the ability to set `clist` be added to `pact -a`?

Reply:
Yes to all of the above, good call.